example/src/screens/App.tsx

```diff
@@ -1,5 +1,5 @@
-import '../../expo-crypto-shim.js';
-import { Alert, SafeAreaView, StyleSheet } from 'react-native';
+// import '../../expo-crypto-shim.js';
```

Comment:
Is this comment still needed? if so perhaps add a `TODO`

Reply:
Thanks for pointing this! Apparently expo now works without that shim 🎉  i'll remove it